.github/workflows/build.yml

```diff
@@ -3,25 +3,21 @@ on:
   pull_request:
   merge_group:
   schedule:
-    - cron: '15 9 * * 1'  # 9:15 UTC weekly on Monday
+    - cron: "15 9 * * 1" # 9:15 UTC weekly on Monday
   workflow_dispatch:
 env:
-    IMAGE_NAME: config
-    IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  IMAGE_NAME: config
+  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
 
 jobs:
-  push-ghcr:
-    name: Build and push image
-    runs-on: ubuntu-24.04
-    permissions:
-      contents: read
-      packages: write
-      id-token: write
+  calculate-tags:
```

Comment:
opted to create a new job for this and reference the tag outputs. It was a little easier structure wise to iterate over the tags and just append the arch when needed